client/src/pages/Dashboard.js

```diff
@@ -1,13 +1,8 @@
 import React from 'react';
-
-import NavBar from '../components/layout/NavBar';
+import AppLayout from './layouts/AppLayout';
 
 const Dashboard = () => {
-  return (
-    <div>
-      <NavBar />
-    </div>
-  );
+  return <AppLayout />;
```

Comment:
In the future are there other things we will be adding to this Dashboard other than the AppLayout? Otherwise I thought the AppLayout would just be the Dashboard then.

Reply:
Yes, this is exactly the case! If a page wants to use a layout, we just need to wrap our component with a layout component.